tests/foreman/cli/test_leapp_client.py

```diff
@@ -25,7 +25,7 @@
 
 RHEL7_VER = '7.9'
 RHEL8_VER = '8.9'
-RHEL9_VER = '9.4'
+RHEL9_VER = '9.3'
```

Comment:
You downgraded RHEL9 version and then never even used the variable, is this a fix for something unrelated to the other changes?

Reply:
We do use it in parameterize the test.